### PR TITLE
feat(Conductor): notify Conductor of new users via webhook

### PR DESCRIPTION
**Pull request by jakeaturner**
Requires two new environment variables:

1. CONDUCTOR_WEBHOOK_BASE=https://commons.libretexts.org/api/v1/central-identity/webhooks
2. CONDUCTOR_API_KEY=

### CODE REVIEW

File: server/controllers/AuthController.ts [hunk 1]
```diff
@@ -1024,6 +1024,43 @@ export class AuthController {
...
+    try {
+
+      const conductorWebhookURL = process.env.CONDUCTOR_WEBHOOK_BASE + '/user-library-access' || 'http://localhost:5000/api/v1/central-identity/webhooks/user-library-access';
+
+      const payload = {
+        central_identity_id: user.uuid,
+        library
+      }
+
+      const res = await fetch(conductorWebhookURL, {
```
Comment (ethanaturner) [inline server/controllers/AuthController.ts:1045]: convert to axios

**Review by ethanaturner — CHANGES_REQUESTED** [review]
see one change, otherwise LGTM

**Comment by jakeaturner** [comment]
ENV config updated

File: server/controllers/UserController.ts [hunk 2]
```diff
@@ -458,26 +448,6 @@ export class UserController {
...
   }
-
-  /**
-   * Retrieves a User's Sandbox URL on a library application.
-   *
-   * @param req - Incoming API request.
-   * @param res - Outgoing API response.
-   * @returns The fulfilled API response.
-   */
-  public async getUserLibraryAppSandboxURL(req: Request, res: Response): Promise<Response> {
```
Comment (ethanaturner) [inline server/controllers/UserController.ts:469]: Keep this until we fully deprecate sandboxes

File: server/models/UserApplication.ts [hunk 3]
```diff
@@ -26,9 +26,6 @@ export class UserApplication extends Model {
   @Column(DataType.INTEGER)
   declare application_id: number;
 
-  @Column(DataType.STRING)
```
Comment (ethanaturner) [inline server/models/UserApplication.ts:29]: Keep this until we fully deprecate sandboxes

File: server/controllers/LibraryController.ts [hunk 4]
```diff
@@ -358,37 +358,6 @@ export class LibraryController {
     }
   }
 
-  /**
-   * Retrieves a library user's Sandbox URL (publicly shareable).
-   */
-  public async getLibraryUserSandboxURL(lib: string, uuid: string) {
```
Comment (ethanaturner) [inline server/controllers/LibraryController.ts:364]: Keep this until we fully deprecate sandboxes

File: server/controllers/LibraryController.ts [hunk 5]
```diff
@@ -159,112 +159,112 @@ export class LibraryController {
   /**
    * Creates a sandbox for a new library user. Assumes that no sandbox exists yet for the user.
    */
-  public async createLibraryUserSandbox(lib: string, libUserID: string, userData: CreateLibraryUserSandboxUserDataInput, headers?: LibraryAPIRequestHeaders) {
```
Comment (ethanaturner) [inline server/controllers/LibraryController.ts:162]: Double-check that there are no other calls to this function (e.g. an API route)

**Review by ethanaturner — CHANGES_REQUESTED** [review]

**Review by ethanaturner — APPROVED** [review]
_ET_. 12/30/23.

**Pull request merged**